pom.xml

```diff
@@ -119,6 +119,12 @@
                         </goals>
                         <configuration>
                             <minimizeJar>true</minimizeJar>
+                            <relocations>
+                                <relocation>
+                                    <pattern>org.apache.commons.cli</pattern>
+                                    <shadedPattern>org.gradoop.examples.thirdparty.commons-cli</shadedPattern>
```

Comment:
A `-` is not allowed as part of a Java identifier, see [Java Specification](https://docs.oracle.com/javase/specs/jls/se12/html/jls-3.html#jls-3.8)
While the JVM should not care about this, is could lead to problems when using related classes.
I think this should be `org.gradoop.examples.thirdparty.commons_cli` or `org.gradoop.examples.thirdparty.org.apache.commons.cli`